### ISSUE TITLE
[UX] MinWidth should be defined

### ISSUE THREAD

**Issue by niels9001** [microsoft/WinUI-Gallery#1181]
### Which version of the app?

WinUI 3 Gallery

### Description

The current implementation allows for extreme minimal widths, breaking the entire UI.

Should be a MinWidth (e.g 480?) to avoid this behavior.

### Screenshots

![image](https://user-images.githubusercontent.com/9866362/214639857-27ca0345-14b3-4bf3-be20-9e294f268b65.png)

**Comment by gregwoo-microsoft** [comment]
@niels9001  Did the Toolkit already implement code to fix this? Wondering if we can port it over to this app.

**Comment by chingucoding** [comment]
I'm afraid the best way to do this currently is to do this: https://github.com/microsoft/microsoft-ui-xaml/issues/2945#issuecomment-776446712

The Toolkit Sample app is still UWP it seems so they have it a bit easier.

**Comment by Mysterious-Dev** [comment]
WinUI Ex makes it easier to set the minimum size. But then it's an extra dependency.

https://github.com/dotMorten/WinUIEx/

**Comment by Jay-o-Way** [comment]
I always would recommend **against** a third-party dependency. **Especially** for an app like this. This should be as native as possible.

**Comment by Jay-o-Way** [comment]
is Subclassing  an option?
